publish_jenkins_console.py

```diff
@@ -7,11 +7,13 @@
 from collections import OrderedDict
 
 import requests
+import urllib
 from requests.exceptions import ConnectionError
 from jenkinsapi.jenkins import Jenkins
 
-# fpast url
-FPASTE_URL = 'https://paste.fedoraproject.org/api/paste/submit'
+APIKEY = urllib.urlencode({'apikey': '5uZ30dTZE1a5V0WYhNwcMddBRDpk6UzuzMu-APKM38iMHacxdA0n4vCqA34avNyt'})
```

Comment:
@FilipB do not hard code API key, Pass it as an environment variable

Reply:
It's hardcoded in fpaste so I guess it will not change: https://pagure.io/fpaste/blob/master/f/fpaste#_34